app/Providers/MediawikiServiceProvider.php

```diff
@@ -53,4 +81,47 @@ public function register(): void
             return null;
         });
     }
+    
+    /**
+     * Register ActionApi implementation
+     */
+    protected function registerActionApi(): void
+    {
+        $this->app->bind(ActionApi::class, function() {
+            if (!$this->isWikiIntegrationEnabled()) {
+                return null;
+            }
+            
+            try {
+                $apiUrl = env('WIKI_URL').'/api.php';
+                $auth = new UserAndPassword(env('WIKI_APIUSER'), env('WIKI_APIPASSWORD'));
+                return new ActionApi($apiUrl, $auth);
+            } catch (\Exception $ex) {
+                Log::error('Failed to create ActionApi for dependency injection: '.$ex->getMessage());
+                return null;
+            }
+        });
+    }
+    
+    /**
+     * Check if wiki integration is enabled
+     */
+    protected function isWikiIntegrationEnabled(): bool
+    {
+        return env('FEATURE__WIKI_INTEGRATION') === true;
+    }
+    
+    /**
+     * Get the services provided by the provider.
+     *
+     * @return array
+     */
+    public function provides()
+    {
+        return [
+            MediawikiFactory::class,
+            UserCreator::class,
+            ActionApi::class,
+        ];
+    }
```

Comment:
Just curious, why was this added?

Reply:
Cause I didn't catch this 🙃. This isn't needed and was added as a debugging method.

I'll go ahead and remove it.